### PR TITLE
Front-End Routes

### PR DESCRIPTION
**Pull request by fares98**
- install react-router-dom
- Add headings for all the components
- Use react-router-dom inside the app

Relates #6

### CODE REVIEW

File: src/routes/routesFolder/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const LANDING = '/';
```
Comment (Fatmasiam) [inline src/routes/routesFolder/index.js]: the folder name just be `constants/ ` and the file name `routes`

Comment (Fatmasiam) [reply src/routes/routesFolder/index.js]: and move it to the `src/` root

**Review by AlaaTaima — DISMISSED** [review]

**Review by yosefanajjar — DISMISSED** [review]

**Review by alaa-yasin — DISMISSED** [review]

File: src/constants/routes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const LANDING = '/landing';
+export const HOME = '/home';
+export const SIGN_IN = '/sign-in';
+export const SIGN_UP = '/sign-up';
+export const FORGET_PASSWORD = '/forget-password';
```
Comment (alaa-yasin) [inline src/constants/routes.js]: forgot-password

**Review by alaa-yasin — DISMISSED** [review]

**Review by Fatmasiam — COMMENTED** [review]
good job 
just edit the `typo` :arrow_up:

File: src/routes/App/index.js [hunk 3]
```diff
@@ -1,11 +1,47 @@
 import React from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import * as ROUTES from '../../constants/routes';
+
+import Landing from '../../containers/Landing';
```
Comment (Fatmasiam) [inline src/routes/App/index.js]: **WHY** we put index file on the containers?!

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**